### ISSUE TITLE
Make GitHub repo names case insensitive

### ISSUE THREAD

**Issue by parshap** [NerdWalletOSS/shepherd#49]
Opening PRs sometimes fails due to missing base branch:

```
[nerdwallet/interstitial-client] 3/3
$ node $SHEPHERD_MIGRATION_DIR/pr-message.js
Step "node $SHEPHERD_MIGRATION_DIR/pr-message.js" exited with 0
✔ Generated PR message
{ HttpError: Empty value for parameter 'base': undefined
    at values.forEach (/Users/ppourkhomami/projects/nerdwallet/shepherd2/node_modules/@octokit/rest/lib/plugins/endpoint-methods/validate.js:80:15)
    at Array.forEach (<anonymous>)
    at Object.keys.forEach.parameterName (/Users/ppourkhomami/projects/nerdwallet/shepherd2/node_modules/@octokit/rest/lib/plugins/endpoint-methods/validate.js:54:12)
    at Array.forEach (<anonymous>)
    at validate (/Users/ppourkhomami/projects/nerdwallet/shepherd2/node_modules/@octokit/rest/lib/plugins/endpoint-methods/validate.js:32:31)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:160:7)
  name: 'HttpError',
  code: 400,
  status: 'Bad Request',
  headers: undefined }
✖ Failed to create pull request
```

It looks like `defaultBranch` is undefined here: https://github.com/NerdWalletOSS/shepherd/blob/46878f22520feb5784700bb16c53b1ed08260ec4/src/adapters/github.ts#L163

**Comment by parshap** [comment]
Figured out why this was happening.

This was because I specified a `--repos` option using a lowercase org name, (e.g., `--repos nerdwallet/interstitial-client`) whereas `repos.json` has the canonical org name casing (`NerdWallet/interstitial-client`).

It'd be nice to make `--repos` case insensitive when pulling data from `repos.json` since GitHub org names are case insensitive.

**Comment by parshap** [comment]
Fixed by @nwalters512 in #192!

**Comment by nwalters512** [comment]
Ha, this is exactly what Taylor ran into! Glad this is fixed now.
